apps/addons/models.py

```diff
@@ -508,12 +512,15 @@ def initialize_addon_from_upload(cls, data, is_listed=True):
         if not locale_is_set:
             addon.default_locale = to_language(trans_real.get_language())
 
+        addon.save()
+        if old_guid_addon:
+            old_guid_addon.update(guid='guid-reused-by-pk%s' % addon.pk)
+            old_guid_addon.save()
         return addon
 
     @classmethod
     def create_addon_from_upload_data(cls, data, user=None, **kwargs):
         addon = cls.initialize_addon_from_upload(data, **kwargs)
-        addon.save()
```

Comment:
It sets `addon.admin_review` after so I was cautious and left the second save in. Should I remove?